Dockerfile

```diff
@@ -55,15 +55,24 @@ FROM pre-build AS build
 # Expose our source so we can install Forseti Security.
 COPY --chown=forseti:forseti . ${WORK_DIR}
 
-RUN pip install --no-cache-dir --upgrade -r requirements.txt --user
+COPY requirements.txt /
+RUN pip install --no-cache-dir --upgrade  --user -r /requirements.txt
```

Comment:
Copy the requirements file into the image and install from there. This will speed up the build time locally, and will only re-build if files have changed locally.